test/fixtures/input/.editorconfig

```diff
@@ -0,0 +1,20 @@
+# EditorConfig helps developers define and maintain consistent
```

Comment:
Thoughts on adding `ember-cli` as a devDep and generating this dynamically? The vast majority of these files do not need to be in the repo, we could generate a new app and then only talk about the "diff" between what we added.

Anyways, I think this is a good follow up PR, we should not block landing on this...

Reply:
I was thinking about this, and would like to. My main concerns are properly generating the `output` application, and the ability to add arbitrary test scenarios in the integration app. I'd like to write more complex scenarios that involve telemetry out in the integration test. So, we'd need some way to generate most of the app, and then copy the app folder over.

Probably we could just do that, just check in the app folders and generate/compare them. I'll work on that.

Reply:
@pzuraq - Let's make an issue to track that, I'd like to circle back around and this is good enough for now...